latex/accessors.tex

```diff
@@ -1052,37 +1139,15 @@ \subsubsection{Image accessor interface}
     image specialization}
   {table.members.accessor.image}
   \addRow
-    { size_t get_count() const }
-    {
-      Returns the number of elements of the SYCL \codeinline{image} this SYCL \codeinline{accessor} is accessing.
-    }
-  \addRow
-    { range<dimensions+1> get_range() const }
+    { size_t get_size() const }
     {
-      Available only when:
-      \codeinline{accessTarget == access::target::image_array)}.
-      \newline
-      Returns a \codeinline{range} object which represents the number of
-      elements of \codeinline{dataT} per dimension that this
-      \codeinline{accessor} may access.
-      \newline
-      The \codeinline{range} object returned must equal to the
-      \codeinline{range} of the \codeinline{image} this \codeinline{accessor}
-      is associated with.
+      Returns the size in bytes of the SYCL \codeinline{image} this SYCL
+      \codeinline{accessor} is accessing.
     }
   \addRow
-    { range<dimensions> get_range() const }
+    { size_t get_count() const }
     {
-      Available only when:
-      \codeinline{accessTarget != access::target::image_array)}.
-      \newline
-      Returns a \codeinline{range} object which represents the number of
-      elements of \codeinline{dataT} per dimension that this
-      \codeinline{accessor} may access.
-      \newline
-      The \codeinline{range} object returned must equal to the
-      \codeinline{range} of the \codeinline{image} this \codeinline{accessor}
-      is associated with.
+      Returns the number of elements of the SYCL \codeinline{image} this SYCL \codeinline{accessor} is accessing.
```

Comment:
After looking again at it, this part looks pretty weird.
Perhaps a merge or rebase that went wild?
I do not think you want to change anything in the area.
Probably the simpler is to add a commit with a copy-paste the content from line 1055 to 1085 of `SYCL-1.2.1/master` to recover the real content.

Reply:
I think these were introduced together with the other unintentional removals and I just missed them while reverting those. Thank you for noticing!